postBuild

```diff
@@ -4,3 +4,9 @@ MPLBACKEND=Agg python -c "import matplotlib.pyplot"  # Build font cache.
 
 # Enable Jupyter extension(s):
 jupyter serverextension enable --py nbgitpuller --sys-prefix
+
+# Create symlink to caproto-repeater to eliminate caReapter message
+ln -s caproto-repeater /srv/conda/bin/caRepeater
+
+# alias using -q to compress unnecessary logging of caproto
+alias caRepeater="caRepeater -q"
```

Comment:
@mrakitin Do you think this has any effect?

Reply:
Apparently it does. Via the Jupyter terminal:

```sh
$ which caRepeater
/srv/conda/bin/caRepeater
```

Reply:
Oops --- I spoke too soon. I think in fact it has no effect.

```
$ caRepeater -q  # no output, as expected
$ caRepeater  # has output, so the alias must not be working
[I 19:06:34.804 repeater:203] Another repeater is already running; exiting
```

@ke-zhang-rd I think threading `-q` through here is not very important, so for the purposes of this PR we can just remove this `alias` and move on if you like. Or, if you want to persevere, we could try my other suggestion,  something like this:

```
echo "/srv/conda/bin/caproto-repeater -q" > /srv/conda/bin/caRepeater
```

instead of the link.

Reply:
I like @danielballan's last solution. Not sure aliases work well enough in scripts. We may try some functions as well, but the two solutions suggested above should be enough.